### ISSUE TITLE
In DataTemplate Labels is invalid

### ISSUE THREAD

**Issue by AuroraDysis** [beto-rodriguez/Live-Charts#86]
New a project and paste following code will get this error.
![image](https://cloud.githubusercontent.com/assets/13709836/12843426/442c73a2-cc34-11e5-9316-444f978eeaa9.png)
```Xml
<StackPanel>
        <liveCharts:PieChart Height="100" Width="100" LegendLocation="Right">
            <liveCharts:PieChart.Series>
                <liveCharts:PieSeries Title="Web Browser Usage" Values="15, 10, 40, 55" DataLabels="True"/>
            </liveCharts:PieChart.Series>
            <liveCharts:PieChart.AxisX>
                <liveCharts:Axis Labels="Explorer, Opera, Mozilla, Chrome"></liveCharts:Axis>
            </liveCharts:PieChart.AxisX>
        </liveCharts:PieChart>
        <ListBox>
            <ListBox.ItemTemplate>
                <DataTemplate DataType="system:String">
                    <liveCharts:PieChart Height="100" Width="100" LegendLocation="Right">
                        <liveCharts:PieChart.Series>
                            <liveCharts:PieSeries Title="Web Browser Usage" Values="15, 10, 40, 55" DataLabels="True"/>
                        </liveCharts:PieChart.Series>
                        <liveCharts:PieChart.AxisX>
                            <liveCharts:Axis Labels="Explorer, Opera, Mozilla, Chrome"></liveCharts:Axis>
                        </liveCharts:PieChart.AxisX>
                    </liveCharts:PieChart>
                </DataTemplate>
            </ListBox.ItemTemplate>
            <system:String>hello</system:String>
            <system:String>hello</system:String>
            <system:String>hello</system:String>
        </ListBox>
    </StackPanel>
```

**Comment by beto-rodriguez** [comment]
Thanks for reporting I am testing and working on it.

**Comment by beto-rodriguez** [comment]
I tested this and I cant see why it is not working first it hits the LabelsValueConverter

https://github.com/beto-rodriguez/Live-Charts/blob/master/LiveChartsCore/TypeConverters/LabelsTypeConverter.cs#L40

and it seems that everything was set correctly, but then labels are null here

https://github.com/beto-rodriguez/Live-Charts/blob/master/LiveChartsCore/CoreComponents/Chart.cs#L960

I can´t see what is going wrong, any help is appreciated!

**Comment by gregsdennis** [comment]
Probably not the cause, but why are you checking `axis.Labels` for null, then just using that or null explicitly? Seems like the check should be to see if `axis` is null.

**Comment by beto-rodriguez** [comment]
@gregsdennis but axis can not be null unless you set it, axis.labels could and should be null if you are using a formatter.

**Comment by AuroraDysis** [comment]
I add a PropertyChangedCallBack
![image](https://cloud.githubusercontent.com/assets/13709836/12956808/ca166b38-d064-11e5-8a0c-728b0b839b6e.png)
and get the result
![image](https://cloud.githubusercontent.com/assets/13709836/12956833/ed712852-d064-11e5-945b-653d7cd6630a.png)
![image](https://cloud.githubusercontent.com/assets/13709836/12956896/3d1c4f76-d065-11e5-97b1-a887a14323be.png)
It seems the property must be changed correctly, but not update.

**Comment by beto-rodriguez** [comment]
@AuroraDysis interesting...

**Comment by luarca84** [comment]
I have a similar problem in LineChart binding Labels. I cant see the labels, only I can see 0,1,2,etc.

`<lvc:LineChart Grid.Row="3" Name="Chart" LegendLocation="Left" Series="{Binding Series}">
                <lvc:LineChart.AxisY>
                    <lvc:Axis Title="Calificaciones"/>
                </lvc:LineChart.AxisY>
                <lvc:LineChart.AxisX>
                    <lvc:Axis Labels="{Binding Labels}" Title="Alumnos" Color="Black">
                    </lvc:Axis>
                </lvc:LineChart.AxisX>
            </lvc:LineChart>`

**Comment by beto-rodriguez** [comment]
@luarca84 can I see your code behind? so I can track this issue easier. Thanks for reporting btw it helpt a lot

**Comment by luarca84** [comment]
The error was mine, now i can see the labels. But I have found a bug of type infinite loop in RadarChart. You can check the code here:
[https://github.com/luarca84/SamplesLiveCharts](https://github.com/luarca84/SamplesLiveCharts)

**Comment by beto-rodriguez** [comment]
Radar chart is not ready yet, it should work for 0.4.x verison or less, but I havent updated to 0.6 a really big update.

**Comment by beto-rodriguez** [comment]
I don´t recommend you to go back to 0.5 or less, 0.6 is much direfent and docs will not match, when I release 0.7 I will fix radar chart please see #79

**Comment by beto-rodriguez** [comment]
@AuroraDysis 0.6.5 has a new syntax and this bug is fixed, please try 0.6.5 synstax is a litle but different only for pie chart, let me know if this is not working for you.
